app/assets/javascripts/mobile/mobile_drawer.js

```diff
@@ -0,0 +1,34 @@
+(function(){
+  Diaspora.Mobile.Drawer = {
+    allAspects: function(){
+      return $("#all_aspects")
+    },
+    followedTags: function(){
+      return $("#followed_tags")
+    },
+    menuBadge: function(){
+      return $("#menu-badge")
+    },
```

Comment:
There was probably a reason, at some point. Now I guess I can directly use the jQuery in the `initialize`.